media_softlet/cmake/linux/media_compile_flags_linux.cmake

```diff
@@ -56,8 +56,6 @@ set(MEDIA_COMPILER_FLAGS_COMMON
     -m${ARCH}
 
     # Global defines
-    -DLINUX=1
```

Comment:
And here. Please, avoid dropping these defines.

Reply:
Thanks, done.